nodejs/dist/uws.js

```diff
@@ -500,6 +500,8 @@ class Server extends EventEmitter {
             this.httpServer.removeListener('upgrade', this._upgradeListener);
         }
 
+        if(this.httpServer) this.httpServer.close()
```

Comment:
Please try to protect code style of the project you contribute to. Compare this to the if-statement above.